### DISCUSSION TITLE
Bottom Navigation

### DISCUSSION THREAD

**Discussion by imanmalekian31** [saadeghi/daisyui#868] [Ideas / feature requests]
Hey,
I had an Idea about `bottom-navigation` and realized that nobody had talked about it so far.
I would like to add it :)

**Comment by imanmalekian31** [comment]
I create it and now you can see it. I would like to know what you think about it.
https://github.com/imanmalekian31/daisyui/tree/feat/bottom-navigation

**Reply by saadeghi** [reply]
Awesome! I will check it soon
